pimd/pim6_cmd.c

```diff
@@ -1516,6 +1516,37 @@ DEFPY (debug_pimv6_zebra,
 	return CMD_SUCCESS;
 }
 
+DEFPY (debug_mroute6,
+       debug_mroute6_cmd,
+       "[no] debug mroute6",
+       NO_STR
+       DEBUG_STR
+       DEBUG_MROUTE6_STR)
+{
+	if (!no)
+		PIM_DO_DEBUG_MROUTE;
```

Comment:
Btw, is this going to enable debugging for v4 version too?

Reply:
No, since the daemons are different.